components/policies/production/base/konflux-rbac/konflux-support-viewer-access/generate-support-viewer-rolebinding-clusterpolicy.yaml

```
@@ -0,0 +1,50 @@
+---
+# This ClusterPolicy automatically generates a RoleBinding in all tenant namespaces
+# to grant read-only access to the 'konflux-sre' and 'ai-konflux-user-support' groups.
+#
+# This policy is designed to ensure that these groups have consistent
+# visibility across tenant namespaces for monitoring, troubleshooting, or support.
+
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: generate-konflux-support-read-only-rolebinding
+  annotations:
+    policies.kyverno.io/title: "Generate Read-Only RoleBinding for Konflux support and sre Groups"
+    policies.kyverno.io/category: Multi-Tenancy
+    policies.kyverno.io/description: >-
+      This policy automatically generates a RoleBinding in all tenant namespaces.
+      The RoleBinding binds the 'konflux-sre' and 'ai-konflux-user-support'
+      groups to the Konflux-specific 'konflux-viewer-user-actions' ClusterRole,
+      granting them comprehensive read-only access to resources within each tenant namespace,
+      therefore allowing better, fast and streamlined support.
+spec:
+  background: true
+  rules:
+    - name: generate-read-only-rolebinding
+      match:
+        any:
+        - resources:
+            kinds:
+            - Namespace
```

let's use the verbose `Group/Version/Kind` format. In some clusters we have a `Namespace` CRD that sometimes confused Kyverno's validation when admitting the ClusterPolicy.

```suggestion
            - /v1/Namespace
```

cf. https://github.com/redhat-appstudio/infra-deployments/blob/0ea9afc8e9d8e4043a0f679099dfba8339dc33ae/components/policies/production/base/namespace-lister/deny-virtual-domain/deny-virtual-domain.yaml#L15